projects/addon-doc/src/components/example/example.template.html

```diff
@@ -42,27 +42,15 @@ <h3 *ngIf="heading" class="t-title">
             </button>
         </div>
 
-        <ng-container
-            *ngIf="{
-                activeTab: tabs[activeItemIndex],
-                code: content[tabs[activeItemIndex]] || ''
-            } as state"
-        >
-            <div
-                *ngIf="state.activeTab == defaultTab; else codeSection"
-                automation-id="tui-doc-example"
-                class="t-demo"
+        <!-- dynamic re-render tab content in runtime for trigger css animation -->
+        <ng-container *ngFor="let tab of tabs; let index = index">
+            <tui-tab-content
+                *ngIf="index === activeItemIndex"
+                [code]="content[tabs[activeItemIndex]]"
+                [isActiveTab]="tabs[activeItemIndex] == defaultTab"
```

Comment:
`===`. Doesn't ESLint highlight that? Also the name is misleading, it's not active tab, it's default tab being active. Why do you go over the tabs in `ngFor` and pass content inside it? Looks wrong.

Reply:
`Why do you go over the tabs in ngFor and pass content inside it?` 

Because that content is re-created every time when was change index and the fade animation will work. I can rename `isActiveTab` to `isPreview`